doc/MediaSigning.xml

```diff
@@ -58,12 +58,6 @@
       signing feature, and also how an ONVIF client should validate the authenticity of a signed
       stream. Audio is outside of scope.</para>
   </chapter>
-  <chapter xml:id="chapter_spec_version">
-    <title>Specification version v1.0</title>
```

Comment:
Note that if we remove this and relay on the Specification revision (e.g., 25.12) only it can take 6 months before a change can be made in the source code.
Example: A change PR is made to this spec in July 2025. It is merged in August 2025 (to the **development** branch). The 25.12 does still not exist, hence strictly speaking the source code should not be updated to reflect the spec until January 2026. That is an awkwardly long time to wait. Or can we see the **development** branch as being the draft for next release? It is publicly available. If so, the source code should be fine to update right after merge.

Reply:
Another concern I have with using the specification revision is that if only a small grammar correction is made to the specification, the revision is updated. So even if the bitstream is intact a change from say 27.06 to 27.12 has no impact on what is used on the camera.

Or even worse, is the revision updated every release even if no change is made?

If the version is to be kept as in 24.12 I propose to change the name to "Bitstream version"

Reply:
ONVIF releases two specifications per year. If there is an urgent need for an intermediate release nothing hinders a `27.11` :-).
A bitstream version would be needed whenever there is no other backward compatible way to parse the new version as e.g. MP4 does with their version fields.

Reply:
If intermediate releases can be made I'm happy with that part. For the concern that tools have to be updated to have the latest version even if a typo has been changed is not a big deal since releases are made only twice a year.